release notes/v0.38.0.md

```diff
@@ -0,0 +1,273 @@
+
+We changed the behavior of how k6 treats panic, which can happen during the execution of JavaScript test scripts code. Previously, the behavior was to catch the panic and log it as an error. Starting with **v0.38.0**, whenever k6 observes a panic, it logs an error like before, but more importantly, it will abort the script execution. This change should help identify and address the issues earlier.
+
+### Miscellaneous
+
+* To achieve consistency with the existing `--log-output` command-line flag, the `--logformat` flag is now considered deprecated and the new `--log-format` takes its place ([2410](https://github.com/grafana/k6/pull/2410)). The same goes for the `K6_LOGFORMAT` environment variable, which is now deprecated and replaced by `K6_LOG_FORMAT` to be consistent with the existing `K6_LOG_OUTPUT`.
+* [2411](https://github.com/grafana/k6/pull/2411) The k6 command-line UI (logo, test description and progerss bars) can now effectively be disabled using the [`--quiet`](https://k6.io/docs/using-k6/options/#quiet) flag.
+* [2411](https://github.com/grafana/k6/pull/2411) The logs can now be saved to a file, or sent to Loki, or completely disabled with [`--log-output`](https://k6.io/docs/using-k6/options/#log-output) flag.
+* [2411](https://github.com/grafana/k6/pull/2411) the end-of-test summary can be sent to a remote server or to file through [`handleSummary()`](https://k6.io/docs/results-visualization/end-of-test-summary/#handlesummary-callback).
+* [2429](https://github.com/grafana/k6/pull/2429) `lib/types` now exposes the source of the `NullHostnameTrie` to simplify the access to an original list of the hostnames.
+
+## Bugs Fixed!
+
+* [2456](https://github.com/grafana/k6/pull/2456) Fixed a very unlikely panic involving arrival rate executors and execution segments with very low VU counts.
+* [2349](https://github.com/grafana/k6/pull/2349) Thanks to @rainingmaster it is now possible to leave the options' tlsAuth `domains` property empty.
+
+
+## Known issues
+
+* @hecnavsanz [reported to us](https://github.com/grafana/k6/issues/2500) that in certain scenarios, k6's UI might inacurately report a test as failed :x: when it actually succeeded :white_check_mark:. This is caused by some concurrency issue in our codebase that only affects the report's UI, and has no impact on the actual test result. A [fix](https://github.com/grafana/k6/pull/2502) is in the work, but won't be ready for this version. We expect to ship it with the next k6 release instead.
+
+## Maintenance
+
+### Dependencies
+
+* [2479](https://github.com/grafana/k6/pull/2479) updated k6's Goja version to `9037c2b61cbf`.
+* [2449](https://github.com/grafana/k6/pull/2449), [2446](https://github.com/grafana/k6/pull/2446), [2444](https://github.com/grafana/k6/pull/2444), and [2443](https://github.com/grafana/k6/pull/2443) updated k6 Go dependencies to their latest compatible versions.
+ 
+### Miscelaneous
```

Comment:
We already have the `Miscellaneous` section, I guess we want to merge them.

Reply:
This is a subsection of the `## Maintenance`. But point taken, I've renamed this one to `### Other` 😸